src/findbugs/excludesFilter.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<FindBugsFilter>
+	<Match>
+		<Class name="jenkins.plugins.slack.webhook.GetProjectLogCommand" />
+		<Method name="execute" />
+		<Bug pattern="DLS_DEAD_LOCAL_STORE" />
+	</Match>
+	<Match>
```

Comment:
Both issues below seem to be real issues. Having a follow-up to clean it up would help

Reply:
Agreed, actually having a follow-up to move the Slack webhook into another plugin would be even nicer. This feature has caused a tremendous amount of issues.